### DISCUSSION TITLE
facing same https://github.com/jarun/nnn/issues/1059 issue in fedora.

### DISCUSSION THREAD

**Discussion by rustiever** [jarun/nnn#1568] [Q&A] [answered]
im using `NAME="Fedora Linux"
VERSION="37 (Workstation Edition)"
`
can you pls help me to compile the nnn??

**Comment by N-R-K** [comment, marked as answer]
You'll have to install the proper dependencies and the headers. For fedora it seems like the correct packages are `readline` and `readline-devel`. https://packages.fedoraproject.org/pkgs/readline/readline/

**Comment by rustiever** [comment]
Now when i run this command `$ make O_NERD=1` i get this error
`cc -DNERD -DICONS_INCLUDE=\"icons-generated-nerd.h\" -std=c11 -Wall -Wextra -Wshadow -O3 -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=600   -o nnn  src/nnn.c -lfts -lreadline -lncursesw -ltinfo  -lpthread
/usr/bin/ld: cannot find -lfts: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [Makefile:210: nnn] Error 1


`

**Reply by N-R-K** [reply]
No clue what that's about. You will have to ask for help in a forum/chat-room for your distro. People there are probably going to be help you with distro specific issues like this.

**Comment by jarun** [comment]
> /usr/bin/ld: cannot find -lfts: No such file or directory

You need to install the fts library.

**Reply by rustiever** [reply]
> ❯ dnfs fts
Last metadata expiration check: 1 day, 1:58:54 ago on Monday 23 January 2023 09:22:03 AM.
======================================================================== Name & Summary Matched: fts =========================================================================
dovecot-fts-xapian.x86_64 : Dovecot FTS plugin based on Xapian
fts-rest-client.noarch : File Transfer Service (FTS) -- Python3 Client and CLI
qca-softstore.i686 : Softstore plugin for the Qt4 Cryptographic Architecture
qca-softstore.x86_64 : Softstore plugin for the Qt4 Cryptographic Architecture
webfts.noarch : Web Interface for FTS
============================================================================= Name Matched: fts ==============================================================================
R-fts.x86_64 : R Interface to 'tslib' (a Time Series Library in C++)
qca-qt5-softstore.i686 : Pkcs11 plugin for the Qt5 Cryptographic Architecture
qca-qt5-softstore.x86_64 : Pkcs11 plugin for the Qt5 Cryptographic Architecture
============================================================================ Summary Matched: fts ============================================================================
avl.x86_64 : Aerodynamic and flight-dynamic analysis of rigid aircrafts
ghc-kan-extensions.x86_64 : Kan extensions, Kan lifts, the Yoneda lemma, and (co)density (co)monads
wildmidi.x86_64 : Softsynth midi player


couldn't find the library @jarun

**Reply by jarun** [reply]
https://packages.fedoraproject.org/pkgs/fts/fts-libs/
